src/Components/QuestionComponentContainer/QuestionComponentContainer.css

```
@@ -75,3 +76,22 @@ p {
   font-size: 1.25rem;
   margin-bottom: 8px;
 }
+
+/* Mobile responsive typography */
+@media (max-width: 767px) {
```

Stylelint (17.2.0)

[error] Expected "context" media feature range notation (media-feature-range-notation)

(media-feature-range-notation)